profiler/__init__.py

```diff
@@ -1,18 +1,20 @@
-from . import controllers  # noqa
-import openerp
+# coding: utf-8
+import logging
+import os
 from cProfile import Profile
+
+import odoo
+from odoo.http import JsonRequest
+from odoo.service.server import ThreadedServer
+
+from . import controllers  # noqa
```

Comment:
Be cause controllers lib is not used, but if controllers is removed, then a `Not found error` is triggered
![image](https://cloud.githubusercontent.com/assets/10233975/23732376/59670e66-0438-11e7-9f7d-6db7a0fe8786.png)

I don't know why this happens yet